### DISCUSSION TITLE
Bug: cannot select node within init script

### DISCUSSION THREAD

**Discussion by mattyod** [eslint/eslint#18468] [General] [answered]
### Environment

Node version: 20.13.1
npm version: 10.5.2
Local ESLint version: n/a 
Global ESLint version: n/a
Operating System: iOS 14.4

### What parser are you using?

Default (Espree)

### What did you do?

```js
npm init @eslint/config@latest
```
Then selected Node as my environment.

### What did you expect to happen?

Node to be selected as my option.

### What actually happened?

Browser was selected.

<img width="706" alt="Screenshot 2024-05-17 at 5 59 45 PM" src="https://github.com/eslint/eslint/assets/457771/c898f56b-bd21-4027-816c-c4d2ea1c2db9">

And
```js
{languageOptions: { globals: globals.browser }}
```
Was added to my `eslint.config.mjs` file.

### Link to Minimal Reproducible Example

http://n/a

### Participation

- [X] I am willing to submit a pull request for this issue.

**Comment by Rec0iL99** [comment, marked as answer]
Hi @mattyod, you can select/de-select using the `<space>` key. The tick icon on the side should turn green if you select an environment.

**Reply by mattyod** [reply]
Aah ok, it wasn't obvious to me that browser and node where checkbox style options rather than select choices like the previous questions. Using space to toggle them on and off does seem to work.
